api/s2n.h

```diff
@@ -421,6 +421,97 @@ extern int s2n_connection_set_client_auth_type(struct s2n_connection *conn, s2n_
 S2N_API
 extern int s2n_connection_get_client_cert_chain(struct s2n_connection *conn, uint8_t **der_cert_chain_out, uint32_t *cert_chain_len);
 
+/**
+ * Sets the initial number of session tickets to send after a handshake.
+ *
+ * @param config A pointer to the config object.
+ * @param num The number of session tickets that will be sent.
+ */
+S2N_API
```

Comment:
out of scope for this pr but i wonder if its possible to leverage another macro to mark which apis are relevant for tls1.2 vs 1.3.

S2N_API_12
vs
S2N_API_13

Reply:
Yeah it gets complicated pretty quickly, because we've got TLS1.2 APIs, TLS1.3 APIs, and APIs that work for both versions. Also maybe someday TLS1.4... 🙃 I'm not sure there is a good solution.